google-cloud-storage/test/helper.rb

```diff
@@ -68,7 +68,9 @@ def random_bucket_hash name: random_bucket_name,
                          rpo: "DEFAULT",
                          autoclass_enabled: nil,
                          autoclass_terminal_storage_class: nil,
-                         enable_object_retention: nil
+                         enable_object_retention: nil,
+                         effective_time: DateTime.now,
+                         retention_duration_seconds: 604800
```

Comment:
I'm assuming this number indicates a 7 day retention period? Maybe append a comment like you did for acceptance tests:

```rb
retention_duration_seconds: 604800 # 7 days
```